examples/source/style-layout/Dark_Mode_Toggle.html

```diff
@@ -0,0 +1,150 @@
+<!---
+- author: tomayac
+- formats
+  - websites
+  - stories
+--->
+
+<!--
+  ## introduction
+
+  todo: This sample demonstrates how to create a dark mode toggle.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="{{filename}}">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>{{title}}</title>
+  <style amp-custom>
+  /* default styles, please remove any you're not using */
+  :root {
+    --color-text-light: #fff;
+    --color-text-dark: #000;
+    --color-bg-light: #FAFAFC;
+    --color-bg-dark: #111;
+    --heading-color: red;
+    --text-color: --color-text-dark;
+    --background-color: --color-text-light;
```

Comment:
Maybe add a comment that explains that this is the default background color. I'd also have expected for this to have `color-bg-light` as value.